packages/schematics/angular/guard/index.ts

```diff
@@ -7,14 +7,35 @@
  */
 
 import { Rule, SchematicsException } from '@angular-devkit/schematics';
+
 import { generateFromFiles } from '../utility/generate-from-files';
+
 import { Implement as GuardInterface, Schema as GuardOptions } from './schema';
 
 export default function (options: GuardOptions): Rule {
-  if (!options.implements) {
-    throw new SchematicsException('Option "implements" is required.');
+  if (!options.implements && !options.guardType) {
```

Comment:
If non is set we should fallback to `CanActivate` to avoid the breaking change. As with this change `implements` becomes as required option which previously was not.

Reply:
This change didn't update the schema for `implements`. I don't think this would be a breaking change here.

Reply:
Oh, you're right. It did :)